app/backend/src/couchers/db.py

```diff
@@ -287,13 +287,14 @@ def can_moderate_at(session, user_id, shape):
     """
     Returns True if the user_id can moderate a given geo-shape (i.e., if the shape is contained in any Node that the user is an admin of)
     """
-    cluster_ids = list(
+    cluster_ids = [
+        cluster_id for (cluster_id,) in
         session.query(Cluster.id)
         .join(Node, Node.id == Cluster.parent_node_id)
         .filter(Cluster.is_official_cluster)
         .filter(func.ST_Contains(Node.geom, shape))
         .all()
-    )
+    ]
```

Comment:
Did this end up being needed? It seems to work on the eastern hemisphere?

Reply:
On my machine I needed it. For me it resulted in an error when checking for moderating rights in the function `_can_moderate_any_cluster` for the query-filter `.filter(ClusterSubscription.cluster_id.in_(cluster_ids))`. The previous list of cluster IDs was a list of tuples like `[(3,), (5,), (11,)]` which the `in_` function would collapse on as the elements are no known basic data type.